### PR TITLE
switch base os to raspberry pi os

### PR DESCRIPTION
**Pull request by dr-bonez**

### CODE REVIEW

File: build/write-image.sh [hunk 1]
```diff
@@ -72,13 +74,13 @@ sudo cp -R frontend/dist/ui /tmp/eos-mnt/var/www/html/main
...
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/00-embassy
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/90-updates-available
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/95-hwe-eol
+#sudo sed -i 's/ENABLED=1/ENABLED=0/g' /tmp/eos-mnt/etc/default/motd-news
+#sudo rm /tmp/eos-mnt/etc/update-motd.d/50-landscape-sysinfo
+#sudo cp ./build/00-embassy /tmp/eos-mnt/etc/update-motd.d
+#sudo chmod -x /tmp/eos-mnt/etc/update-motd.d/*
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/00-embassy
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/90-updates-available
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/95-hwe-eol
```
Comment (dr-bonez) [inline build/write-image.sh:83]: IDK what the raspbian equivalent of this is,  so I've disabled it for now

@k0gen

File: build/partitioning.sh [hunk 2]
```diff
@@ -3,4 +3,4 @@
 set -e
 
 # Use fdisk to create DOS partition table with 4 primary partitions, set 1 as bootable, write, and quite
-(echo o; echo n; echo p; echo 1; echo 2048; echo 526335; echo t; echo c; echo n; echo p; echo 2; echo 526336; echo 1050623; echo t; echo 2; echo c; echo n; echo p; echo 3; echo 1050624; echo 16083455; echo n; echo p; echo 16083456; echo 31116287; echo a; echo 1; echo w) | sudo fdisk ${OUTPUT_DEVICE}
+(echo o; echo x; echo i; echo "0xcb15ae4d"; echo r; echo n; echo p; echo 1; echo 2048; echo 526335; echo t; echo c; echo n; echo p; echo 2; echo 526336; echo 1050623; echo t; echo 2; echo c; echo n; echo p; echo 3; echo 1050624; echo 16083455; echo n; echo p; echo 16083456; echo 31116287; echo a; echo 1; echo w) | sudo fdisk ${OUTPUT_DEVICE}
```
Comment (Blu-J) [inline build/partitioning.sh:6]: Interesting

**Review by Blu-J — APPROVED** [review]

File: build/partitioning.sh (hunk 2 above)
Comment (dr-bonez) [reply build/partitioning.sh:6]: the core issue seemed to be that raspios didn't want to mount the rootfs by label. so, instead, we mount by PARTUUID, which is based on the MBR disk identifier which we set here.

File: Makefile [hunk 3]
```diff
@@ -69,4 +70,4 @@ patch-db/client/dist: $(PATCH_DB_CLIENT_SRC) patch-db/client/node_modules
 	npm --prefix patch-db/client run build
 
 # this is a convenience step to build all frontends - it is not referenced elsewhere in this file
-frontend: frontend/node_modules $(EMBASSY_UIS) 
\ No newline at end of file
+frontend: frontend/node_modules $(EMBASSY_UIS)
```
Comment (ProofOfKeags) [inline Makefile:73]: Wat?

**Review by ProofOfKeags — APPROVED** [review]

File: Makefile (hunk 3 above)
Comment (dr-bonez) [reply Makefile:73]: trailing newline

File: build/write-image.sh (hunk 1 above)
Comment (k0gen) [reply build/write-image.sh:83]: Done #1205

**Pull request merged**